pkg/relayer/proxy/proxy.go

```diff
@@ -140,10 +144,15 @@ func (rp *relayerProxy) Start(ctx context.Context) error {
 	for _, relayServer := range rp.servers {
 		server := relayServer // create a new variable scoped to the anonymous function
 
-		// Ensure that each backing data node responds to a ping request
-		// (at least) before continuing operation.
-		if err := server.Ping(ctx); err != nil {
-			return err
+		// Only test the connectivity of the backing data nodes if pingEnabled is true.
+		if rp.pingEnabled {
+			// Ensure that each backing data node responds to a ping request
+			// (at least) before continuing operation.
+			if err := server.Ping(ctx); err != nil {
+				rp.logger.Error().Err(err).
+					Msg("failed to ping backend service before starting relay server")
```

Comment:
Can you provide some more context in the log message.

Which backend server? ServiceID? Host? Etc.

Reply:
Since the the `server` (interface) does not contain such information. I updated the `Ping` implementation to include the failing `serviceID` and `backendUrl`.